contributors/index.js

```diff
@@ -14,6 +14,14 @@ const list = [{
 		github_username: 'sethusenthil',
 		twitter_username: 'sethusenthilll',
 	},
+	{
```

Comment:
@ramanujamgond Please don't add your details to the middle of the list. New details should be added to the bottom of the list so that the website can show the latest contributors in order.

Reply:
Thank you for the response @RocktimSaikia - Just give me a moment I will update and again raise a PR

Reply:
@RocktimSaikia - I have made the changes kindly review the PR

Reply:
LGTM :+1: